src/se/edu/inclass/task/Deadline.py

```diff
@@ -0,0 +1,8 @@
+from task.Task import Task 
+
+class Deadline(Task):
```

Comment:
deadline would have a field to indicate the due date (in the fix branch) and mechanism to populate and display the new field

Reply:
for fix branch - taken note